package.json

```diff
@@ -14,10 +14,12 @@
     "start:scheduler:prod": "NODE_ENV=production node schedule.js",
     "setup": "node scripts/setup.js",
     "lint": "./node_modules/eslint/bin/eslint.js ./test/**/*.js ./src/**/*.js ./services/**/*.js",
+    "lint:fix": "./node_modules/eslint/bin/eslint.js ./test/**/*.js ./src/**/*.js ./services/**/*.js --fix",
```

Comment:
One can simply run `npm run lint -- --fix`, I'm not in favor of exposing this option in a script.